api/errors.go

```diff
@@ -47,3 +47,29 @@ func NewErrorInternal(msg string) error {
 func NewErrorUnimplemented() error {
 	return newErrorGRPC(codes.Unimplemented, "not implemented")
 }
+
+// HTTP Mapping: 503 Service Unavailable
+// Unavailable is used instead of 500 to indicate to the client that it can retry the operation.
+// See the documentation for the FAILED_PRECONDITION error code in https://grpc.io/docs/guides/status-codes/
+// which compares FAILED_PRECONDITION, UNAVAILABLE, and ABORTED.
+func NewErrorUnavailable(msg string) error {
+	return newErrorGRPC(codes.Unavailable, msg)
+}
+
+// HTTP Mapping: 503 Service Unavailable
+// NewErrorUnavailableWithRetry is like NewUnavailableError, but allows the caller to specify a retry delay.
```

Comment:
since it is not used, should we remove it?

Reply:
think it doesn't hurt to keep around, especially if we *might* use it in the future, since it's non-trivial code to write, so its good to keep the example around I feel.